src/analysisfunctions.jl

```diff
@@ -0,0 +1,37 @@
+function _locreg(df, xaxis::LinSpace, x, y; kwargs...)
+    model = Loess.loess(convert(Array{Float64,1},df[x]),convert(Array{Float64,1},df[y]); kwargs...)
```

Comment:
Actually, I can imagine that some of the columns could be integers with a very wide range (e.g. some time given in milliseconds) and the user would still want to use Loess regression which however doesn't accept arrays of integers, so I thought it was better to leave it as is.